src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -207,9 +250,29 @@ def _rbd_list(self, pool_name=None):
         for pool in pools:
             # pylint: disable=unbalanced-tuple-unpacking
             status, value = self._rbd_pool_list(pool)
-            for i, image in enumerate(value):
-                value[i]['configuration'] = RbdConfiguration(pool, image['name']).list()
             result.append({'status': status, 'value': value, 'pool_name': pool})
+
+        ts, disk_usage, ex = Cache.get('rbd_disk_usage')
+        if ex:
+            logger.debug("failed to calulate images disk usage: %s", ex)
```

Comment:
`s/calulate/calculate/`

...and shouldn't this be, at least, a `warn` instead of `debug`?